xarray/test/test_plot.py

```diff
@@ -293,6 +295,11 @@ def test_center(self):
         self.assertIsNone(cmap_params['levels'])
         self.assertIsNone(cmap_params['norm'])
 
+    def test_norm(self):
+        cmap_params = _determine_cmap_params(self.data,
+                                             norm=mpl.colors.SymLogNorm(0.1))
+        self.assertIsNotNone(cmap_params['norm'])
+
```

Comment:
Ideally, we can test this without having to test a private function like `_determine_cmap_params`.  We should be able to check that the facet grid subplots are using the correct `norm`.